Vizzuality/landgriffon
# Change table style

### General description

This PR updates the table style 
### Designs

https://www.figma.com/file/7bmF29CIXDJwCpuloW5Nt6/%F0%9F%94%B5-Landgriffon---Redesign_BLUE?type=design&node-id=3310-113219&t=qdkS8WyTQQw5CISi-4

### Testing instructions

Go to /analysis/table and check if the style (default and expanded rows) and pagination are aligned with the designs

### Related task
[LANDGRIF-1299](https://vizzuality.atlassian.net/browse/LANDGRIF-1299)

## Checklist before merging



[LANDGRIF-1299]: https://vizzuality.atlassian.net/browse/LANDGRIF-1299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
